get_all_manifests.sh

```diff
@@ -20,6 +20,8 @@ declare -A COMPONENT_MANIFESTS=(
     ["trainingoperator"]="opendatahub-io:training-operator:dev:manifests"
     ["datasciencepipelines"]="opendatahub-io:data-science-pipelines-operator:main:config"
     ["modelcontroller"]="opendatahub-io:odh-model-controller:incubating:config"
+    ["feastoperator"]="opendatahub-io:feast:master:infra/feast-operator/config"
```

Comment:
```suggestion
    ["feastoperator"]="opendatahub-io:feast:0.42.0:infra/feast-operator/config"
```
can we try pointing to a versioned tag instead?

Reply:
we'll still want to wait for the next release, 0.43.0 before merging this.

Reply:
here, we try using  a floating branch so we get new code all the time, only switch to a tag from component when  release odh.

Reply:
we have created the `stable` branch and updated accordingly.